crates/sui-core/src/consensus_handler.rs

```diff
@@ -11,11 +11,12 @@ use narwhal_types::ConsensusOutput;
 use std::collections::hash_map::DefaultHasher;
 use std::hash::{Hash, Hasher};
 use std::sync::{Arc, Mutex};
-use sui_types::base_types::AuthorityName;
+use sui_types::base_types::{AuthorityName, EpochId};
 use sui_types::messages::ConsensusTransaction;
 use tracing::{debug, instrument, warn};
 
 pub struct ConsensusHandler {
+    _epoch: EpochId,
```

Comment:
I don't understand where this will be used, since you aren't using it yet. I assume the idea is that we want to make sure that it is equal to `state.epoch()`, and if it isn't, we know that reconfig has happened. But where will that occur?

Reply:
nevermind, i see your next PR now